expression/generator/other_vec.go

```diff
@@ -193,6 +192,7 @@ func (b *{{.SigName}}) vecEvalInt(input *chunk.Chunk, result *chunk.Column) erro
 		{{- end }}
 		for i := 0; i < n; i++ {
 			if r64s[i] != 0 {
+				result.SetNull(i, false)
```

Comment:
if `len(args)` is 1, this line will not be run.
I think we should set null to false when `b.hashSet` is hit.
```go
if _, ok := b.hashSet[arg0]; ok {
    r64s[i] = 1
    result.SetNull(i, false)
}
```